portfolio/src/main/webapp/script.js

```diff
@@ -56,22 +65,92 @@ async function getServerContent() {
  */
 function createCommentElement(comment) {
   const divElement = document.createElement('div');
-  divElement.setAttribute("class", "comment-div");
+  divElement.setAttribute('class', 'comment-div');
   
-  var userP = document.createElement("span");
+  const userP = document.createElement("span");
   userP.innerText = comment.user; 
-  userP.setAttribute("class", "user-attr");
+  userP.setAttribute('class', 'user-attr');
 
-  var dateP = document.createElement("span");
+  const dateP = document.createElement('span');
   dateP.innerText = comment.commentDate; 
-  dateP.setAttribute("class", "date-attr");
+  dateP.setAttribute('class', 'date-attr');
+
+  const deleteButton = document.createElement('button');
+  deleteButton.innerHTML = '<i class="fas fa-trash-alt"></i>';
+  deleteButton.setAttribute('class', 'del-attr');
+  deleteButton.addEventListener('click', () => {
+    deleteComment(comment);
+  });
 
-  var textP = document.createElement("p"); 
+  const textP = document.createElement('p');
   textP.innerText = comment.content;
-  textP.setAttribute("class", "content-attr");
+  textP.setAttribute('class', 'content-attr');
 
   divElement.appendChild(userP);
+  divElement.appendChild(deleteButton);
   divElement.appendChild(dateP); 
   divElement.appendChild(textP); 
   return divElement;
 }
+
+/**
+ * Clears div element containing comments.
+ */
+function clearDiv() {
+  const commentEl = document.getElementById('comment-container');
+  while(commentEl.firstChild) {
+    commentEl.removeChild(commentEl.firstChild);
+  }
+}
+
+/**
+ * Deletes all comments from Datastore.
+ */
+async function deleteAll() {
+  const response = await fetch('/delete-data', {method: 'POST'});
+  getServerContent();
+}
+
+/**
+ * Deletes specified comment.
+ */
+async function deleteComment(comment) {
+  const params = new URLSearchParams();
+  params.append('id', comment.id);
+  const response = await fetch('/delete-data', {method: 'POST', body: params});
+  getServerContent();
+}
+
+/**
+ * Changes text of selected number option then retrieves server content.
+ */
+function changeDisplayNum() {
+  const selectEl = document.getElementById('display-num');
+  var index = selectEl.selectedIndex;
+  for(var i = 0; i < selectEl.options.length; i++) {
+    if(index === i) {
+      selectEl[i].innerHTML = 'Show: ' + selectEl[i].innerHTML;
```

Comment:
This is a lot of logic to control including the word 'Show' for the default option in the list. Perhaps you can communicate that this option controls what gets shown in the list outside the dropdown option? You could then delete this logic and the function below it.